WordPress/src/main/java/org/wordpress/android/ui/activitylog/detail/ActivityLogDetailFragment.kt

```diff
@@ -74,12 +79,21 @@ class ActivityLogDetailFragment : Fragment() {
                 uiHelpers.setTextOrHide(activityActorRole, activityLogModel?.actorRole)
 
                 val spannable = activityLogModel?.content?.let {
-                    notificationsUtilsWrapper.getSpannableContentForRanges(it, activityMessage, { range ->
-                        viewModel.onRangeClicked(range)
-                    }, false)
+                    notificationsUtilsWrapper.getSpannableContentForRanges(
+                            it,
+                            activityMessage,
+                            { range ->
+                                viewModel.onRangeClicked(range)
+                            },
```

Comment:
Minor (🔍): Consider making this lambda a one-liner: `{ range -> viewModel.onRangeClicked(range) },`

Reply:
@ParaskP7 - this was an auto AS thing. We should review settings as a team during our next sync